x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_permissions.ts

```diff
@@ -49,112 +50,114 @@ export async function storedPackagePoliciesToAgentPermissions(
     return;
   }
 
```

Comment:
For reviewer: Remove whitespace to improve readability of this diff: https://github.com/elastic/kibana/pull/166234/files?diff=unified&w=1